packages/scandipwa/src/component/ProductBundleItems/ProductBundleItems.container.js

```diff
@@ -73,42 +78,94 @@ export class ProductBundleItemsContainer extends ProductCustomizableOptionsConta
         this.setState({ isLoading: false });
     }
 
-    getOptionPrice(item, selectedValues) {
-        const { option_id } = item;
-        let price = 0;
-        let priceExclTax = 0;
+    optionPriceMap = {
+        [PRICE_TYPE_DYNAMIC]: this.getDynamicOptionPrice.bind(this),
+        [PRICE_TYPE_FIXED]: this.getFixedOptionPrice.bind(this),
+        [PRICE_TYPE_PERCENT]: this.getPercentOptionPrice.bind(this)
+    };
+
+    getDynamicOptionPrice(product, quantity) {
+        const {
+            price_range: {
+                minimum_price: {
+                    regular_price: {
+                        value: itemPrice = 0
+                    } = {},
+                    final_price: {
+                        value: finalItemPrice = 0
+                    } = {},
+                    final_price_excl_tax: {
+                        value: finalItemPriceExclTax = 0
+                    } = {}
+                } = {}
+            } = {}
+        } = product;
+
+        const optionInitialPrice = itemPrice * quantity;
+        const optionPrice = finalItemPrice * quantity;
+        const optionPriceExclTax = finalItemPriceExclTax * quantity;
```

Comment:
Please provide comments on this in the code. What is "initial price"? What is the other price?